panther_manager/include/panther_manager/plugins/action/undock_robot_action_node.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 Open Navigation LLC
```

Comment:
Please apply all possible comments from DockRobot to files related to UndockRobot